src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -688,18 +690,18 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
       </div>
       {templateID === VenueTemplate.themecamp && (
         <div style={{ textAlign: "center" }}>
-          You'll be able to add rooms to your theme camp on the next page
+          You&apos;ll be able to add rooms to your theme camp on the next page
         </div>
       )}
       {formError && (
         <div className="input-error">
           <div>One or more errors occurred when saving the form:</div>
-          {Object.keys(errors).map((fieldName) => (
-            <div>
+          {Object.keys(errors).map((fieldName, index) => (
+            <div key={index}>
```

Comment:
[NB, nit] I think this would be fixed better by using `fieldName`? (assuming we only report a single error per field?)